src/services/NodeReducerService.ts

```
@@ -281,100 +255,152 @@ export class NodeDupReduction {
+              // Uncheck if want to log similar nodes
+              // clogger.info('Found similar node', {
+              //   sourceId,
+              //   targetId: similarNode.nodeId,
+              //   similarity: similarNode.similarity,
+              // });
+
+              alts.push({ id: similarNode.nodeId, name: similarNode.nodeName });
+
+              if (!this.verbose) {
+                await this.migrateNodeRelations(sourceId, similarNode.nodeId);
+                if (nodesObj.hasOwnProperty(similarNode.nodeId)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)